### DISCUSSION TITLE
Note Icon Display Issue: Visible in Edit Mode, Missing in Read Mode

### DISCUSSION THREAD

**Discussion by matisync** [FlorianWoelki/obsidian-icon-folder#400] [Q&A] [answered]
I have a note with an icon. I made a link to that note from another note. The link shows the note icon in edit mode but not in read mode.

**Comment by FlorianWoelki** [comment, marked as answer]
This is a known issue. I've created an issue for that: https://github.com/FlorianWoelki/obsidian-iconize/issues/409

**Reply by matisync** [reply]
Thank you
